tests/Commands/FlushCommandTest.php

```diff
@@ -21,4 +24,23 @@ public function it_will_clear_the_cache()
 
         $this->assertDifferentResponse($firstResponse, $secondResponse);
     }
+
+    /** @test */
+    public function it_will_fire_events_when_clearing_the_cache()
+    {
+        $mock = $this->getMockBuilder('stdClass')->setMethods(['callback'])->getMock();
+
+        $mock->expects($this->exactly(2))
+            ->method('callback');
+
+        Event::listen(FlushingResponseCache::class, function () use ($mock) {
```

Comment:
There are dedicated methods with which you can assert if an event was sent. For some examples take a look at the tests of laravel-uptime-monitor.